applications/FluidDynamicsApplication/custom_utilities/cfd_utilities.cpp

```diff
@@ -0,0 +1,442 @@
+//    |  /           |
```

Comment:
I am a bit lost. You are adding a lot of unrelated stuff in here no?

Reply:
There are few unrelated stuff like `EvaluateInPoint`, `CalculateConditionGeometryData`, which are copies taken from elements and conditions. I couldn't find a central place where they are implemented for reuse, each time we define them in the elements or conditions. :/

`CalculateConditionNormal` method is there in NormalCalculataionUtils, but it assigns nodal historical value NORMAL as well which is problematic. That is because if these methods are used along with wall boundaries, and there is an adjacent SLIP boundary, using the method in NormalCalculationUtils will overwrite NORMALs of common nodes to wall boundaries and SLIP boundaries with wall boundary normals. :/